app/routes/wallet/dids.py

```diff
@@ -82,12 +129,28 @@ async def get_public_did(
     return result.result
 
 
-@router.put("/public", response_model=DID)
+@router.put("/public", response_model=DID, summary="Set Public DID")
 async def set_public_did(
     did: str,
     auth: AcaPyAuth = Depends(acapy_auth_from_header),
 ) -> DID:
-    """Set the current public DID."""
+    """
+    Set the current public DID
+    ---
+
+    This endpoint allows you to set the current public DID.
+    The tenant needs a connection with an endorser to make a did public and by default
+    only issuers will have public DIDs and so only issuers can update their public did.
```

Comment:
Run-on sentence, and mixes capital DIDs with lower case dids. Just refactor this a lil.